main-page/index.html

```diff
@@ -10,20 +10,39 @@
 <main>
 <header>
     <div>
-        <h1>Wild Pokémon have Appeared</h1>
+        <span>Wild Pokémon have Appeared</san>
     </div>
 </header>
 <body>
     <Section class="main-section">
+        
         <label>
+            <div> 
+            <p class="banner"></p>
             <input type="radio" name="pokemon"><img /></input>
-        </label>
+            <div class="encountered"><h1></h1></div>
+            </div>  
+        </label> 
         <label>
+            <div>
+            <p class="banner"></p>    
             <input type="radio" name="pokemon"><img /></input>
+            <div class="encountered"><h1></h1></div>
+            </div>
         </label>
+        
         <label>
+            <div> 
+             <p class="banner"></p> 
             <input type="radio"  name="pokemon"><img /></input>
+            <div class="encountered"><h1></h1></div>
+            </div>
         </label>
+        <audio class="my_audio" id="audio" controls preload="none">
```

Comment:
so much fun, Justin! Great touch and you should definitely share for others. Audio is such a fun thing to incorporate in apps :)